pkg/scheduler/schedule_one.go

```diff
@@ -142,6 +142,11 @@ func (sched *Scheduler) schedulingCycle(
 ) (ScheduleResult, *framework.QueuedPodInfo, *framework.Status) {
 	logger := klog.FromContext(ctx)
 	pod := podInfo.Pod
+	// TODO(knelasevero): Remove duplicated keys from log entry calls
+	// When contextualized logging hits GA
+	// https://github.com/kubernetes/kubernetes/issues/111672
+	logger = klog.LoggerWithValues(logger, "pod", klog.KObj(pod))
```

Comment:
this should be even higher in the call stack, in `scheduleOne`.

Reply:
Done: https://github.com/kubernetes/kubernetes/compare/d400a5d743fa38b191c0a9ffd30e6d446573486f..2a23061f6c23b65a916956f2ede2371041944b15